omemo/state.py

```diff
@@ -232,7 +235,11 @@ def create_msg(self, from_jid, jid, plaintext):
         # Encrypt the message key with for each of receivers devices
         for rid, cipher in session_ciphers.items():
             try:
-                encrypted_keys[rid] = cipher.encrypt(key).serialize()
+                if self.trust[self.isTrusted(cipher)] is True:
+                    encrypted_keys[rid] = cipher.encrypt(key).serialize()
+                else:
+                    log.warn('Skipped Device because Trust is: ' +
```

Comment:
You should change it to `debug` or change it to `info` but only print it if state is `UNDECIDED`